src/components/calcite-input-date-picker/calcite-input-date-picker.tsx

```diff
@@ -278,19 +272,49 @@ export class CalciteDate {
                   value={formattedDate}
                 />
               </div>
-            )}
-            {this.renderCalendar(activeDate, dir, maxDate, minDate, date, endDate)}
-            {this.range && !this.noCalendarInput && this.layout === "horizontal" && (
+            }
+
+            <div
+              aria-hidden={(!this.active).toString()}
+              class="menu-container"
+              ref={this.setMenuEl}
+            >
+              <div
+                class={{
+                  ["calendar-picker-wrapper"]: true,
+                  ["calendar-picker-wrapper--end"]: this.focusedInput === "end",
+                  [PopperCSS.animation]: true,
+                  [PopperCSS.animationActive]: this.active
+                }}
+              >
+                <calcite-date-picker
```

Comment:
@paulcpederson do these props all make sense? I'm not really certain which need to be passed and which do not.

I'm going to need an experts help that is more familiar with the date component.

---

Why do we have both end and endAsDate as well as start and startAsDate? Could we maybe just have one prop for these that accepts either a string or a `Date()`? Maybe @jcfranco can chime in here?